Jellyfin.Server/StartupOptions.cs

```diff
@@ -15,6 +18,12 @@ public class StartupOptions : IStartupOptions
         [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (database files, etc.).")]
         public string? DataDir { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the server should not host static web content.
+        /// </summary>
+        [Option(ConfigurationExtensions.NoWebContentKey, Required = false, HelpText = "Indicates that the web server should not host any static web content.")]
+        public bool NoWebContent { get; set; }
```

Comment:
I'd change this to `WebContent` and set the default to true

Reply:
I think specifying `--no-web-content` is more in-line with standard command line flags than `--web-content=false`?

Reply:
Maybe, but boolean properties/variables shouldn't be "negative" if you know what I mean. `!WebContent` vs double negation `!NoWebContent`, which one is easier?

Reply:
Yeah, that's a good point I think you're right.

Would it be worse to use `NoWebContent` for the command line flag and `WebContent` for the `IConfiguration` setting that is actually used in the application? The command line property is only used in one place [here](https://github.com/jellyfin/jellyfin/pull/2601/files/7dd2f3a8f2dc6ea4db5ef9e1db4684b39cbcb6d3#diff-14e7bdf45ccd3fc82c0c85043d8d52d7R83) in order to set the value for the `IConfiguration`

Reply:
I like that idea, but I would use the term `web client` instead. When we finalize the split we can also switch the argument to `--web-client` as well.

Reply:
I've renamed the command line switch to `--nowebclient` and changed the configuration setting to `HostWebClient` (which defaults to true)